### DISCUSSION TITLE
firejail tree+list flags bypass hidepid mounted /proc displaying details of other users sandboxes

### DISCUSSION THREAD

**Discussion by ghost** [netblue30/firejail#4703] [Q&A] [answered]
Is this a bug or the intended behavior? It _seems_ like it would be a bug since using firemon with/without the --tree/--list flags fails as expected with the message:
`Error: /proc is mounted hidepid, you would need to be root to run this command`

If it is intentional is there a setting that will "fix" it?

firejail version information(Fedora 34):
```
firejail version 0.9.64.4

Compile time support:
	- AppArmor support is disabled
	- AppImage support is enabled
	- chroot support is enabled
	- D-BUS proxy support is enabled
	- file and directory whitelisting support is enabled
	- file transfer support is enabled
	- firetunnel support is enabled
	- networking support is enabled
	- overlayfs support is disabled
	- private-home support is enabled
	- private-cache and tmpfs as user enabled
	- SELinux support is enabled
	- user namespace support is enabled
	- X11 sandboxing support is enabled
```

**Comment by rusty-snake** [comment, marked as answer]
I converted this to #4705 as it looks like a [CWE-200](https://cwe.mitre.org/data/definitions/200.html) bug.

**Comment by reinerh** [comment]
Is this bug also in 0.9.66?
When I run `firejail --tree` (or `--list`) inside a firejail sandbox I only get the error:
```
Warning: an existing sandbox was detected. /bin/bash will run without any additional sandboxing features
```

**Reply by ghost** [reply]
I don't believe firejail supports being run within a firejail sandbox in ?any? way. I get the same error by just trying to run firejail with the --help flag if it's already in a sandbox.
